### DISCUSSION TITLE
Support the Visual Editor also for standard script files

### DISCUSSION THREAD

**Discussion by albertomercurio** [quarto-dev/quarto-cli#11372] [Feature Requests]
### Description

Quarto already supports rendering [standard script files](https://quarto.org/docs/computations/render-scripts.html) like `.py` or `.jl`, by simply dividing the cells with `# %%`. However, it would be very nice if also the Visual Editor could support such files. In this way, I can still work with a pure `.py` file, but in a visual way, by rendering markdown and Latex equations for example. And when I need to run this file elsewhere (e.g., a cluster), the `.py` file is ready to use.

I find the Visual Editor very game-changing (although it doesn't support Copilot yet), and the support to also standard script file would be awesome.

**Comment by mcanouil** [comment]
Repository for the visual editor is: https://github.com/quarto-dev/quarto

**Reply by albertomercurio** [reply]
Ok, should I move this discussion there?

**Reply by mcanouil** [reply]
Only if it's not already there.
Please do link back here.
